helm-chart/renku-gateway/values.yaml

```diff
@@ -37,6 +37,26 @@ replicaCount: 1
 ## implications and should never be done in a production setting.
 development: false
 
+## To protect the backend services from an excessive amount of API calls
+## issued by one client, one can enforce rate limits here. The limits apply
+## per UI client session (identified by the cookies). For an explanation of
+## the different values check out the rate limiting documentation of traefik
+## v2.0.
+rateLimits:
+  ## General rate limit, applies to all /api calls combined.
+  general:
+    extractorfunc: request.header.cookies
+    period: 10s
+    average: 20
+    burst: 100
+  ## Special rate limit for the notebooks service (/api/notebooks)
+  notebooks:
+    extractorfunc: request.header.cookies
+    period: 10s
+    average: 3
```

Comment:
With `3` here, sometimes I get errors from the server endpoint since it may be invoked every 3 seconds for a while. 

![Screenshot from 2019-06-12 13-23-47](https://user-images.githubusercontent.com/43481553/59348027-697b5c80-8d16-11e9-8309-87e90772175c.png)

4 should be enough, I would go for 5 to be safe -- it's enough to avoid getting errors with a single tab open, but not enough for 2 tabs.

Reply:
Ok, I will move to 5 here 👍